examples/lighting-app/k32w/main/include/AppTask.h

```diff
@@ -89,6 +89,7 @@ class AppTask
 
     Function_t mFunction;
     bool mResetTimerActive;
+    bool mSyncClusterToButtonAction;
```

Comment:
should we initialize members to nullptr/0/false/... ?

Reply:
yes, operated this change. thanks!